tasks/common.yml

```diff
@@ -9,14 +9,15 @@
       state: directory
       owner: "{{ sensu_user_name }}"
       group: "{{ sensu_group_name }}"
-      mode: "0750"
+      mode: "0555"
```

Comment:
Okay, last thing I swear, this change is interesting to me. We originally added it via https://github.com/sensu/sensu-ansible/commit/158e17815f9015f0bd15be0c428016a386ba7577 and then updated it in https://github.com/sensu/sensu-ansible/commit/82fb6e70b5d7a15503adc7b9c5a2ca47cf8b002c to the current value.
I know that I was able to get the integration tests to pass when those were merged. Do you happen to still have the error? (I might be able to pull it fronm the builds.) I'd also say that this should probably be `0755`, right?

Reply:
My error comes from additional molecule testing we have configured which breaks when 'others' doesn't have access. Nothing from the testing within this role, but how other use it after the installation has completed.

I was looking at past installations with this role prior to commit 158e178 had the permissions set to ( What seems to be the default RPM install )
```
$ stat -c "%a %n" conf.d/
755 conf.d/
```

Then compared it with installations from Puppet and its current module settings which configure the permissions at 
https://github.com/sensu/sensu-puppet/blob/master/manifests/init.pp#L360
```
$ stat -c "%a %n" conf.d/
555 conf.d/
```

Having the dir set to 555 prevents anyone but root( or sudo ) from making configuration changes hopefully limiting any hand jamb configuration settings and providing a better audit trail.  Its early and I could totally be missing something do you mind explaining what you see the benefits of having 755 vs 55 are for this directory ?

As as side node the puppet module is very mature and widely used for consistency with the community IMO these roles / modules should look to offer parity as much as possible.

Reply:
Ah yeah, cool, that all makes sense (thought I was crazy since I couldn't repro any build errors on my end). I didn't actually checkout what puppet chose, but I noticed back during that change that `sensu-chef` uses `0750`: https://github.com/sensu/sensu-chef/blob/cc3da414ec6fc55b479bdf8d430908082625293b/attributes/default.rb#L29 

So, I'd like to actually get a standard across the board on things like this to improve consistency between each CM repo. That said, I _think_ we should do here what both Puppet and Chef did and move this setting to a variable to allow folks to override the default. 
As for what the default should be, I feel like 0555 makes the most sense. With the owner set to `sensu`, it should never have to write to this directory (unless that's a thing I'm not aware of), and then anyone can read from this directory. My only concern with the everyone can read part is that then folks have to explicitly ensure that files they create are locked down. Looking at https://github.com/sensu/sensu-ansible/blob/master/tasks/plugins.yml that's not currently a thing. 

So, anyway, I'm game for the default to be `0555` but, ideally we bump this out to a variable for folks to 'chose their own adventure' but, that can be done in a seperate PR. 

Does the above seem fair? (I haven't had much coffee today either)

Reply:
I agree that getting a standard across all the CMs is the ideal path and also about this being a variable for the future.  Interesting that Chef sets it to 750, I wonder what the reasoning behind that is 🤔 

It seems like a small project in itself to verify and convert what is already managed so I like that being in a separate PR.

As far as I know there should be no reason for the sensu user to be writing to the conf.d directory.  I've never had it configured that way :P